RetailCoder.VBE/UI/CodeExplorer/Commands/AddClassModuleCommand.cs

```diff
@@ -15,12 +15,12 @@ public AddClassModuleCommand(AddComponentCommand addComponentCommand) : base(Log
             _addComponentCommand = addComponentCommand;
         }
 
-        protected override bool CanExecuteImpl(object parameter)
+        protected override bool EvaluateCanExecute(object parameter)
         {
             return _addComponentCommand.CanAddComponent(parameter as CodeExplorerItemViewModel);
         }
 
-        protected override void ExecuteImpl(object parameter)
+        protected override void OnExecute(object parameter)
```

Comment:
suggests an event .. is it though? `DoExecute` or even `Execute` seems like the better name to me...